pkg/querier/http_test.go

```diff
@@ -481,118 +321,16 @@ func TestVolumeHandler(t *testing.T) {
 	})
 }
 
-func TestResponseFormat(t *testing.T) {
```

Comment:
note: not removed but moved to the serialize file's test as that's where this logic lives now